src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -1588,19 +1588,15 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "NewVersionFeatures_Info_about_abb_information" = "Änderungen zum Release finden Sie in den App-Informationen unter dem Menüpunkt „Neue Funktionen”.";
 
-/* Version 2.6 */
+/* Version 2.7 */
 
-"NewVersionFeature_26_euCertCheck_title" = "EU-Zertifikatsprüfung";
+"NewVersionFeature_27_sigCheck_title" = "Signaturprüfung";
 
-"NewVersionFeature_26_euCertCheck_description" = "Sie können nun vor einer Reise Zertifikate für ein gewähltes Land auf Gültigkeit prüfen.";
+"NewVersionFeature_27_sigCheck_description" = "Die Signatur Ihrer Zertifikate wird nun beim Hinzufügen in der Corona-Warn-App auf Gültigkeit und Echtheit geprüft. Damit können nur Zertifikate importiert werden, die von einer autorisierten Stelle ausgestellt wurden.";
 
-"NewVersionFeature_26_enhancedStats_title" = "Erweiterung Statistiken";
+"NewVersionFeature_27_technicalValidity_title" = "Prüfung technische Gültigkeit";
```

Comment:
"technische Gültigkeit" is a completely new term and users will be unsure what this means. Maybe this should be explained here (IHMO best through a link to a FAQ article "Was ist die technische Gültigkeit?")

@ UA

Reply:
Hi @Ein-Tim 
Regarding the headlines, thanks for the hint. We already have an own ticket for this. 
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8510

Reply:
Hi @Ein-Tim Thanks for your feedback. we'll discuss it.

> "technische Gültigkeit" is a completely new term and users will be unsure what this means. Maybe this should be explained here (IHMO best through a link to a FAQ article "Was ist die technische Gültigkeit?")
> 
> @ UA

Reply:
@NataliaLemmerth @SwissMountainFrog 

Regarding the link, I opened: #3242

I used @SwissMountainFrog's implementation from back in version 1.14 (link to screenshots section). 

I'm really no Swift expert, so please take a look at it and bear with me if I did something wrong. 😅